rollup/cmd/gas_oracle/app/app.go

```diff
@@ -50,49 +50,40 @@ func action(ctx *cli.Context) error {
 	if err != nil {
 		log.Crit("failed to load config file", "config file", cfgFile, "error", err)
 	}
-	subCtx, cancel := context.WithCancel(ctx.Context)
+	instanceCtx, instanceCancel := context.WithCancel(ctx.Context)
+	loopCtx, loopCancel := context.WithCancel(ctx.Context)
```

Comment:
why create two ctx ?

Reply:
The `instanceCtx` is for managing the lifecycle of instances: `handleConfirmLoop` should be closed after `sender.Stop()`.

The `loopCtx` is for managing the lifecycle of polling (which could send new transactions): `xxxSender.SendTransaction` should be closed first.